src/client/src/components/UserTimeline.jsx

```diff
@@ -155,63 +156,89 @@ const UserTimeline = () => {
                 }}
               ></img>
             </div>
-            
+
             <span className="userNameTimeline">
               {sessionStorage.getItem("firstname")}{" "}
               {sessionStorage.getItem("lastname")}
             </span>
             <br></br>
-            <span className="connectionsLinkTimeline">View Connections</span>
+            <span className="userNameTimeline">
+              <IconButton>
+                <NotificationsNoneIcon></NotificationsNoneIcon>
+              </IconButton>
+            </span>
           </div>
 
           {/* User posts section*/}
           <div className="middleTimeline">
             {/* Create a post section*/}
+            <br></br>
+            <div style={{ fontWeight: "bold", textAlign: "center" }}>
+              Welcome to your timeline{" "}
+            </div>
             <div className="userInformationTimeline">
-              <IconButton>
-                <EditIcon onClick={savePost} fontSize="medium"></EditIcon>
+              <IconButton style={{ marginTop: "10px" }}>
+                <SendIcon onClick={savePost} fontSize="medium"></SendIcon>
               </IconButton>
+
               <TextField
                 id="outlined-basic"
                 label="Start A Post"
                 variant="standard"
                 className="postTextField"
-                value= {postData.description}
-                onChange={e => setpostData({ ...postData, description: e.target.value })}
+                value={postData.description}
+                onChange={(e) =>
+                  setpostData({ ...postData, description: e.target.value })
+                }
+              />
+
+              <div
+                className="timestamp"
+                value={Date.now()}
+                onChange={(e) =>
+                  setpostData({ ...postData, timestamp: e.target.value })
+                }
```

Comment:
It's better to have the message icon at the right of the text field.

Reply:
![image](https://user-images.githubusercontent.com/77070862/225999009-47d5fd04-bb63-452b-941c-ad7fead01d6d.png)